doc-content/drools-docs/src/main/asciidoc/AuthoringAssets/test-scenarios-background-rule-based-proc.adoc

```diff
@@ -0,0 +1,22 @@
+[id='test-scenarios-background-rule-based-proc']
+= Adding a background data in rule-based test scenarios
+
+Follow the procedure below to add and set a background data in rule-based test scenarios.
+
+.Prerequisites
+* The rule-based test scenario template are created for the selected test scenario. For more information about creating rule-based test scenarios, see xref:test-designer-create-test-template-rule-based-proc[].
+* The individual test scenarios are defined. For more information about defining a test scenario, see xref:test-designer-test-scenario-definition-proc[].
+
+.Procedure
+. Open the rule-based test scenarios in the test scenario designer.
+. Click the *Background* tab of the test scenarios designer.
+. Click an instance header cell in the *GIVEN* section to add a background data object field.
+. Select the data object from the *Test Tools* tab.
```

Comment:
Suggestion - change the sentence order to coincide with the rest of the doc:

s/Select the data object from the *Test Tools* tab./From *Test Tools* panel, select the data object.